docs/requirements.txt

```diff
@@ -0,0 +1,5 @@
+pydantic
+httpx
+aiorwlock
+async_property
+readerwriterlock
```

Comment:
What does this file do for us? Why can't we use the requirements listed in setup.cfg?

Reply:
The requirements listed here are duplicative of what's in the setup.cfg file, however this is how ReadtheDocs encourages users to declare them : https://docs.readthedocs.io/en/stable/config-file/v2.html